lib/model_configuration/model_configuration.rb

```diff
@@ -62,11 +65,11 @@ def assign_attributes_from_model_configuration(hash)
     @namespaces = hash[@model_name][:namespaces] || []
     @skip_seeds = hash[@model_name][:skip_seeds] || false
     @skip_ui    = hash[@model_name][:skip_ui] || false
+    @soft_delete = hash[@model_name][:soft_delete].nil? ? true : hash[@model_name][:soft_delete]
```

Comment:
ok - booleans and junk. Have discussed @osiro will change the above to be the same so all is consistent.

Could do something like

```ruby
@soft_delete = configuration_for(:soft_delete, default: true)
```